### ISSUE TITLE
DependencyInjection for ReaderFactory does not work in Symfony 5.2

### ISSUE THREAD

**Issue by posido** [gpslab/geoip2#69]
Hello,
```
Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Cannot autowire argument $readerFactory of "App\Controller\Test\TestController::geoip()": it references class "GpsLab\Bundle\GeoIP2Bundle\Reader\ReaderFactory" but no such service exists." at C:\Projects\testProject\vendor\symfony\dependency-injection\Container.php line 435
Hide context    Hide trace
[▼
  "exception" => Symfony\Component\DependencyInjection\Exception\RuntimeException {#10076 ▶}
]
{▼
  C:\Projects\testProject\vendor\symfony\dependency-injection\Container.php:435 {▼
    Symfony\Component\DependencyInjection\Container->getService($registry, string $id, ?string $method, $load) …
    › if (\is_string($load)) {
    ›     throw new RuntimeException($load);
    › }
  }
...
```
Thanks
Rico

**Comment by peter-gribanov** [comment]
Thank you for contacting us.  The problem is resolved.  The release will be a little later.

Happy New Year!!! 🎉

**Comment by posido** [comment]
Thank you!!!

**Comment by peter-gribanov** [comment]
Fix was released in [2.0.6](https://github.com/gpslab/geoip2/releases/tag/v2.0.6).
